app/src/main/java/com/example/keniphhomemaintenance/ui/cookbook/KeniphOutlinedTextField.kt

```diff
@@ -0,0 +1,41 @@
+package com.example.keniphhomemaintenance.ui.cookbook
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.text.KeyboardActions
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material.OutlinedTextField
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.focus.FocusDirection
+import androidx.compose.ui.focus.FocusManager
+import androidx.compose.ui.text.input.ImeAction
+import androidx.compose.ui.text.input.KeyboardType
+
+@Composable
+fun KeniphOutlinedTextField(
+    textMutableState: MutableState<String>,
+    textLabel: String,
+    focusManager: FocusManager,
+    keyboardAction: KeniphKeyboardActions = KeniphKeyboardActions.DEFAULT
+) =
+
+    OutlinedTextField(
+        modifier = Modifier
+            .fillMaxWidth(),
+        value = textMutableState.value,
+        onValueChange = { textMutableState.value = it },
+        label = { Text(textLabel) },
+        keyboardOptions = KeyboardOptions(
+            keyboardType = KeyboardType.Text,
+            imeAction = ImeAction.Next
+        ),
+        keyboardActions = KeyboardActions(onNext = {
+            when (keyboardAction) {
+                KeniphKeyboardActions.PREVIOUS -> focusManager.moveFocus(FocusDirection.Up)
+                KeniphKeyboardActions.NEXT -> focusManager.moveFocus(FocusDirection.Down)
+                else -> focusManager.clearFocus()
```

Comment:
It's generally not great practice to rely on else for behavior that you will want explicitly - what I mean is that you have a DEFAULT value in your enum, but it will only be handled by the else statement here. We should instead have a KeniphKeyboardActions.DEFAULT branch in this when statement, and then the else statement can just do nothing, since we'll never hit it (or at least, we should never hit it). 

For bonus points, you can implement the Enum as a Sealed Class to eliminate the need for the else statement entirely, but I won't hold up the PR for that

Reply:
AS told me to just remove the else statement even without sealing the enum